src/pybind/mgr/volumes/fs/nfs.py

```diff
@@ -225,6 +225,15 @@ def _create_user_key(self, entity, path, fs_name):
 
         return json_res[0]['entity'], json_res[0]['key']
 
+    def _delete_user(self, entity):
+        ret, out, err = self.mgr.mon_command({
+            'prefix': 'auth del',
+            'entity': 'client.{}'.format(entity),
+            })
+
+        if ret!= 0:
+            log.error(f"User could not be deleted: {err}")
```

Comment:
(a few days ago, I added `check_mon_command` that will raise, if `ret != 0`. Not related to this particular call to mon_command)

Reply:
I will add `check_mon_command`

Reply:
Done